src/layer/tile/GridLayer.js

```diff
@@ -446,11 +451,13 @@ L.GridLayer = L.Layer.extend({
 			this._tiles[key].current = false;
 		}
 
+		if (Math.abs(zoom - this._tileZoom) > 1) { this._setView(center, zoom); }
```

Comment:
Maybe a comment here would help future maintainers (with mantra "say why not what" in mind, of course) ?